packages/katello/rubygem-katello/rubygem-katello.spec

```diff
@@ -28,7 +28,8 @@ Requires: foreman < %{foreman_max_version}
 # start specfile generated dependencies
 Requires: foreman >= %{foreman_min_version}
 Requires: %{?scl_prefix_ruby}ruby(release)
-Requires: %{?scl_prefix_ruby}ruby
+Requires: %{?scl_prefix_ruby}ruby >= 2.5.0
+Requires: %{?scl_prefix_ruby}ruby < 2.7.0
```

Comment:
This line also surprised me but it matches what https://github.com/Katello/katello/commit/2da9d83902bb55b57cfd797b694844d89446193d introduced. If we want to update to the 2.7 SCL it should be updated.

Reply:
i think it was likely not intended, the commit doesn't look intentional, i filed an issue:  https://projects.theforeman.org/issues/31958